### PR TITLE
Deploy storybook.posthog.net

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Github Action to deploy changes on `master` to https://storybook.posthog.net/
- Adds a welcome page (`.mdx`) to storybook
- Hides the "add-ons" panel by default

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

File: .github/workflows/storybook-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    storybook-deployment:
+        # Operating System
+        runs-on: ubuntu-latest
+        # Job steps
+        steps:
+            - name: Check out PostHog/posthog repo
+              uses: actions/checkout@v2
+              with:
+                  fetch-depth: 0
+            - uses: actions/checkout@v1
```
Comment (Twixes) [inline .github/workflows/storybook-deploy.yml:24]: We only need to checkout once.

**Review by Twixes — CHANGES_REQUESTED** [review]

**Comment by Twixes** [comment]
BTW curious why not use `@storybook/storybook-deployer`'s `storybook-to-ghpages` command?

File: .github/workflows/storybook-deploy.yml (hunk 1 above)
Comment (mariusandra) [reply .github/workflows/storybook-deploy.yml]: Doh. That explains another bug...

**Comment by mariusandra** [comment]
> BTW curious why not use `@storybook/storybook-deployer`'s `storybook-to-ghpages` command?

Hmm.. didn't know of it . This felt like something a few unix commands could accomplish, so didn't bother looking for a library 🤷 :).

**Review by Twixes — APPROVED** [review]
Not Unix commands!!!!!11 GNU stands for "GNU's not Unix"!

**Pull request merged**